### DISCUSSION TITLE
Request for publishing the source code of SPA files

### DISCUSSION THREAD

**Discussion by ahadvirus** [vpnhood/VpnHood#297] [General]
Dear VpnHood team.
I try to make some changes in the client's app but found the client's app is a SPA file shown as a client's app with a component.
Could you publish the source code of SPA? or just publish the source code of the CSS files?

**Comment by trudyhood** [comment]
It would be great if anyone can convert it to vue3.
https://github.com/vpnhood/VpnHood.Client.WebUI
